asrtoolkit/file_utils/name_cleaners.py

```diff
@@ -63,5 +65,5 @@ def generate_segmented_file_name(target_dir, file_name, iseg):
     """
     Take a target location, a current location, and a segment number and generate a target filename
     """
-    return target_dir + os.sep + basename(strip_extension(file_name)) + \
-              "_seg_{:05d}.".format(iseg) + file_name.split(".")[-1]
+    return sanitize_hyphens(target_dir + os.sep + basename(strip_extension(file_name)) + \
```

Comment:
So the problem was hyphens weren't handled?

Reply:
Worse than that, they were being handled by the stm file but not for the audio files.